test/e2e/page-objects/pages/account-list-page.ts

```diff
@@ -313,6 +329,20 @@ class AccountListPage {
     }
 
     await this.driver.clickElement(addAccountButton);
+
+    // Run if there are multiple srps
+    if (accountType === ACCOUNT_TYPE.Ethereum && srpIndex) {
+      const srpName = `Secret Recovery Phrase ${srpIndex.toString()}`;
+      // We click twice because, the first click is to go to the SRP List
+      // from there we select the SRP that we want to add the account to
+      await this.driver.clickElement({
+        text: 'Secret Recovery Phrase 1',
+      });
+      await this.driver.clickElement({
```

Comment:
Hopefully I got this right:

```suggestion
      // First, we first click here to go to the SRP List.
      await this.driver.clickElement({
        text: 'Secret Recovery Phrase 1',
      });
      // Then, we select the SRP that we want to add the account to.
      await this.driver.clickElement({
```